### PR TITLE
Change display of run time for bundles

### PR DESCRIPTION
**Pull request by yipenghe**
Addressing #1199 
1. Changed to display run time below the command and positioned to right so it won't be the first information that comes to eye
2. Formatted to hh:mm:ss for run time > 1s, display 0.xxxxxs for run time less than 1 second
2. Truncate the time to show at most 5 decimals, run time smaller than 0.00001s will just show '< 0.00001s' instead
3. Added a clock icon

Screenshot from 2019-10-05 14-24-56
Screenshot from 2019-10-05 14-51-36

### CODE REVIEW

**Comment by yipenghe** [comment]
Formatted run time

**Review by percyliang — CHANGES_REQUESTED** [review]
We should use the same rendering as the logic for rendering durations in `formatting.py`.  We can either pass string versions from the backend, or if that's inconvenient, just have a `Formatting.js` that parallels `formatting.py`.

**Comment by yipenghe** [comment]
> We should use the same rendering as the logic for rendering durations in `formatting.py`. We can either pass string versions from the backend, or if that's inconvenient, just have a `Formatting.js` that parallels `formatting.py`.

Found a exisiting render duration function and used that

File: frontend/src/components/worksheets/BundleDetail/MainContent.jsx [hunk 1]
```diff
@@ -30,21 +32,23 @@ class MainContent extends React.Component<
 		const stateSpecClass = bundleInfo.state === 'failed'
             ? 'failedState'
             : (bundleInfo.state === 'ready' ? 'readyState' : 'otherState');
-		
+        //Get the correct run time display
```
Comment (percyliang) [inline frontend/src/components/worksheets/BundleDetail/MainContent.jsx:36]: nit: space before "Get"

File: frontend/src/components/worksheets/BundleDetail/MainContent.jsx [hunk 2]
```diff
@@ -30,21 +32,23 @@ class MainContent extends React.Component<
 		const stateSpecClass = bundleInfo.state === 'failed'
             ? 'failedState'
             : (bundleInfo.state === 'ready' ? 'readyState' : 'otherState');
-		
+        //Get the correct run time display
+        let bundleRunTime;
```
Comment (percyliang) [inline frontend/src/components/worksheets/BundleDetail/MainContent.jsx]: Simplify: `const bundleRunTime = bundleInfo.metadata.time ? renderDuration(...) : ...`;

**Review by percyliang — APPROVED** [review]

File: frontend/src/components/worksheets/BundleDetail/MainContent.jsx (hunk 2 above)
Comment (yipenghe) [reply frontend/src/components/worksheets/BundleDetail/MainContent.jsx]: Thanks, addressed!

**Pull request merged**